### PR TITLE
[IT] translation week11

### PR DESCRIPTION
**Pull request by fguiso**
11-2.md and 11-3.md (11.md and 11-1.md are already translated)

### CODE REVIEW

File: docs/it/week11/11-2.md [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+             1-\cos(x_1,x_2), & \quad y=1,  \\
+             \max(0,\cos(x_1,x_2)-\text{margine}), & \quad y=-1
+             \end{array}
+   \right.
+$$
+
+Questa funzione di perdita misura la somiglianza fra due input, utilizzando la distanza del coseno di similitudine, e viene tipicamente utilizzata per apprendere immersioni non-lineari o per l'apprendimento semi-supervisionato.
+
+* Definito in maniera alternativa, 1 meno il coseno dell'angolo fra due vettori è praticamente la distanza Euclidea normalizzata.
+* Il vantaggio <!-- svantaggio? --> di questa funzione è che quando si hanno due vettori e li si vuole rendere molto distanti fra di loro, lo si può ottenere semplicemente rendendo i vettori molto lunghi. Ovviamente questo non è ottimale, perché non vogliamo aumentare la lunghezza dei vettori ma ruotarli nella direzione giusta. Per questo, si normalizzano i vettori e si calcola la distanza Euclidea fra di loro.
```
Comment (fguiso) [inline docs/it/week11/11-2.md:196]: L'originale dice "advantage"... ma da ciò che viene descritto sembrerebbe un disadvantage perché poi si dice "this is not optimal."

Frase di riferimento: "The advantage of this is that whenever you have two vectors and you want to make their distance as large as possible, it is very easy to make the network achieve this by make the vectors very long. Of course this is not optimal."

File: docs/it/week11/11-2.md [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+* Per le coppie di vettori "positive" (appartenenti alla stessa categoria), questa perdita tenta di allineare i vettori fra di loro il più possibile. Per le coppie "negative" (appartenenti a categorie diverse), la perdita rende il coseno minore di un margine, che deve avere un valore basso ma positivo.
+* In uno spazio ad alta dimensionalità, abbiamo una concentrazione alta di punti <!--?--> attorno all'"equatore" della sfera. Dopo aver normalizzato i vettori, tutti i punti si trovano distribuiti sulla sfera. Desideriamo che i punti semanticamente simili siano vicini. Gli esempi dissimili, invece, dovrebbero essere ortogonali. Non vogliamo che siano in direzione opposta perché vi è un solo punto al polo opposto <!--?-->. Dato che sull'"equatore" abbiamo una grande quantità di spazio, vorremmo avere un margine dal valore basso cosí da poter sfruttare tutta l'area in questa regione.
```
Comment (fguiso) [inline docs/it/week11/11-2.md:198]: Si può tradurre "there is a lot of **area** near the the equator of the sphere" come "**concentrazione di punti** attorno all'equatore della sfera"? o meglio lasciare "**area**"

File: docs/it/week11/11-2.md [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+
+
+### [Perdita a classificazione temporale connettivista (Temporal Classification, CTC) - `nn.CTCLoss()`](https://www.youtube.com/watch?v=bj1fh3BvqSU&t=4103s)
+
+Calcola la perdita fra una serie temporale continua (non segmentata) e una sequenza di _target_.
+*  La perdita _CTC_ prende la somma sulle probabilità di diversi allineamenti fra la sequenza d'input e la sequenza di _target_. Questo produce un valore di perdita che è differenziabile rispetto a ciascun nodo di input.
+* L'allineamnto dell'input e il target viene presunto essere "_many-to-one_" <!-- come tradurre many-to-one? -->, il che limita la lunghezza della sequenza di target così da renderla minore o uguale alla lunghezza della sequenza d'input.
```
Comment (fguiso) [inline docs/it/week11/11-2.md:224]: come tradurre many-to-one?

File: docs/it/week11/11-3.md [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+<!-- <center>
+<img src="{{site.baseurl}}/images/week11/11-3/figure3.png" style="zoom: 30%; background-color:#DCDCDC;" /><br>
+<b>Figure 3:</b> Illustration of an agent in the world model
+</center> -->
+
+## Dati
+
+<!-- ## Data set -->
+
+Prima di discutere di come apprendere il modello della realtà, esploriamo i dati che abbiamo. Abbiamo 7 telecamere montate sopra ad un palazzo di 30 piani che si affaccia sull'autostrada interstatale. Aggiustiamo le telecamere per ottenere una visuale dall'alto verso il basso _top-down_ e poi estraiamo le "scatole" che definiscono il confine (_bounding box_) di ciascun veicolo. <!-- come tradurre bounding box? --> Nel periodo $t$, possiamo determinare $p_t$ che rappresenta la posizione della macchina, $v_t$ che ne rappresenta la velocità, e $i_t$ che ne rappresenta lo stato corrente del traffico attorno al veicolo.
```
Comment (fguiso) [inline docs/it/week11/11-3.md:74]: come tradurre bounding box?

**Comment by fguiso** [comment]
This week had a few phrases that I wasn't sure how to translate. I pointed these out using comments for the reviewer, but I'm also adding comments here.

**Review by Atcold — APPROVED** [review]
Impressive! Thanks so much!

**Pull request merged**